hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/mod_mean.F

```diff
@@ -609,13 +617,13 @@ subroutine mean_archiv(n, iyear,iday,ihour)
       call zaiowr(u_m(1-nbdy,1-nbdy,k),iu,.true.,
      &            xmin,xmax, nopa, .false.)
       if     (mnproc.eq.1) then
-      write (nop,117) 'utotl   ',nmean,time_ave,k,coord,xmin,xmax
+      write (nop,117) 'u-vel.  ',nmean,time_ave,k,coord,xmin,xmax
```

Comment:
@Jipingnersc: you need to include Alfatih@s update in the velocity names before committing, otherwise you will undo his changes.